frontend/src/components/common/icons/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Icon } from "@chakra-ui/react";
```

Comment:
for some reason, I can never get `@chakra-ui/icons` to resolve (types are there, deps are added) so I just created the components manually based off figma